static/app/views/monitors/components/quickStartEntries.tsx

```diff
@@ -0,0 +1,94 @@
+import {Fragment} from 'react';
+import merge from 'lodash/merge';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
+
+export interface QuickStartProps {
+  dsnKey?: string;
+  orgSlug?: string;
+  slug?: string;
+}
+
+const VALUE_DEFAULTS = {
+  dsnKey: '<my-dsn-key>',
+  orgSlug: '<my-organization-slug>',
+  slug: '<my-monitor-slug>',
+};
+
+function withDefaultProps(props: QuickStartProps): Required<QuickStartProps> {
+  return merge(VALUE_DEFAULTS, props);
+}
```

Comment:
i'm not sure this is worth doing vs `?? <my-dsn-key>` at the top of the function that uses it

Reply:
There will be a number of these little guides.